x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java

```diff
@@ -261,7 +261,7 @@ public void testRollupAfterRestart() throws Exception {
             client().performRequest(bulkRequest);
 
             // create the rollup job
-            final Request createRollupJobRequest = new Request("PUT", "/_xpack/rollup/job/rollup-job-test");
+            final Request createRollupJobRequest = new Request("PUT", "/_rollup/job/rollup-job-test");
```

Comment:
Since this is a full cluster restart bwc test, I think we could go from 6.x to 7.0 right?  Do we need to check if we're on 6.x so we can use the old endpoint? 

BWC test always hurt my brain so maybe not.

Reply:
Yes, that would be right!

Reply:
I pushed 463198a5be73f623b92c7f8c23b7f119cf729ad5.